http-echo.yaml

```diff
@@ -49,8 +32,14 @@ update:
     strip-prefix: v
 
 test:
+  environment:
+    contents:
+      packages:
+        - curl
   pipeline:
-    # AUTOGENERATED
     - runs: |
-        http-echo --version
+        http-echo --version | grep ${{package.version}}
         http-echo --help
+    - runs: |
+        http-echo -listen=:8080 -text="hello world" &
```

Comment:
I'd prefer we use `test/daemon-check-output` here and used `wait-for-it` afterwards to guarantee port availability before using curl